fuse_examples/tutorials/hello_world/hello_world.ipynb

```diff
@@ -0,0 +1,1411 @@
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "torch_model = LeNet()\n",
+    "\n",
+    "model = FuseModelWrapper(model=torch_model,\n",
+    "                        model_inputs=['data.image'],\n",
+    "                        post_forward_processing_function=perform_softmax,\n",
+    "                        model_outputs=['logits.classification', 'output.classification']\n",
+    "                        )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### **Loss function**\n",
+    "Dictionary of loss elements. each element is a sub-class of FuseLossBase.\n",
+    "\n",
+    "The total loss will be the weighted sum of all the elements.\n",
+    "\n",
+    "Fuse's loss API extracts the predications and the labels from the dictonary hierarchy and then applies the callable loss function while considering the weight."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "losses = {\n",
+    "    'cls_loss': FuseLossDefault(pred_name='model.logits.classification', target_name='data.label', callable=F.cross_entropy, weight=1.0),\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### **Metrics**\n",
+    "Dictionary of metric elements. Each element is a sub-class of FuseMetricBase.\n",
+    "\n",
+    "The metrics will be calculated per epoch for both the validation and train.\n",
+    "\n",
+    "The 'best_epoch_source', used to save the best model could be based on one of these metrics."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "metrics = OrderedDict([\n",
+    "    ('operation_point', MetricApplyThresholds(pred='model.output.classification')), # will apply argmax\n",
+    "    ('accuracy', MetricAccuracy(pred='results:metrics.operation_point.cls_pred', target='data.label'))\n",
+    "])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### **Callbacks**\n",
+    "Callbacks are sub-classes of FuseCallbackBase.\n",
+    "\n",
+    "A callback is an object that can preform actions at various stages of training.\n",
+    "\n",
+    "In each stage it allows to manipulate either the data, batch_dict or epoch_results.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "callbacks = [\n",
+    "    FuseTensorboardCallback(model_dir=paths['model_dir']),  # save statistics for tensorboard\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### **Train**\n",
+    "Building Fuse's manager and supplying it PyTorch's optimizer and scheduler.\n",
+    "\n",
+    "The manager is the main API while using Fuse - it resposible for the Train and Infer functionallity.\n",
```

Comment:
Let's remove this line (it's not necessarily the main API)

Reply:
Sure. Also removed it from FuseManagerDefault's documentation :)